README

```diff
@@ -41,32 +41,57 @@ The callback will only report on the status of tasks that have a name beinging w
 
 Note::
     If you haven't deployed using infrared, you can still run the tests if you create your own inventory file, containing one group of hosts called ``overcloud_nodes``.
-
+    Alternatively, you can create your own playbook that imports the tasks in ``tasks/*.yml``
```

Comment:
This line is fine, but the filename should probably be converted to README.md (or README.rst or README.adoc depending on the syntax in use)